app/lib/screens/medical/manager/history/history_manager_detail.dart

```diff
@@ -0,0 +1,448 @@
+                              ),
+                              ElevatedButton(
+                                onPressed: () async {
+                                  if (_formKey.currentState!.validate()) {
+                                    final res =
+                                        await _submitHistoryManagerDetails(
+                                      doctorId: selectedDoctorId,
+                                      patient: widget.student.email,
+                                      details: detailsFieldController.text,
+                                      diagnosis: diagnosisFieldController.text,
+                                      treatment: treatmentFieldController.text,
+                                    );
+
+                                    if (res) {
+                                      await showFeedbackDialog(context,
+                                          title: "Submitted successfully",
+                                          content:
+                                              "Medical History submitted successfully",
+                                          defaultActionText: "Close");
+                                      Navigator.of(context).pop();
+                                    }
+                                  }
+                                },
+                                child: const Text("Add"),
+                              ),
+                              const SizedBox(
+                                height: 10.0,
+                              ),
+                            ],
+                          ),
+                        )
+                      ],
+                    ),
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Future showFeedbackDialog(
+    BuildContext context, {
+    required String title,
+    required String content,
+    required String defaultActionText,
+  }) {
+    return showDialog(
+      context: context,
+      builder: (context) => AlertDialog(
+        content: Column(
+          mainAxisSize: MainAxisSize.min,
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            const SizedBox(height: 10.0),
+            SizedBox(
+                height: 100.0,
+                child: Image.asset("assets/images/icon _tick circle.png")),
+            const SizedBox(height: 20.0),
+            Text(
+              title,
+              style: const TextStyle(
+                fontSize: 18.0,
+              ),
+            ),
+            const SizedBox(height: 10.0),
+            Text(content),
+          ],
+        ),
+        actions: [
+          TextButton(
+            onPressed: () => Navigator.of(context).pop(false),
+            child: Text(defaultActionText),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class DoctorModel {
+  DoctorModel(
+    this.id,
+    this.name,
```

Comment:
Move this class to a separate file. One file should only contain one concern.

Reply:
Okay